api/v1beta1/flinkcluster_types.go

```diff
@@ -560,6 +564,21 @@ type FlinkClusterStatus struct {
 	// The status of savepoint progress
 	Savepoint *SavepointStatus `json:"savepoint,omitempty"`
 
+	// CurrentRevision indicates the version of FlinkCluster.
+	CurrentRevision string `json:"currentRevision,omitempty"`
```

Comment:
Add more information about this field, e.g., why it is a string, how it is generated.

Reply:
Done.